parley/src/style/mod.rs

```diff
@@ -22,6 +22,14 @@ pub enum WhiteSpaceCollapse {
     Preserve,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum LineHeight {
+    /// Line height specified as a multiple of the font size.
```

Comment:
Is a relative line height ever affected by the selected font (not just now, but is that something we might theoretically ever want?). That to me would be the only reasonable argument for keeping a relative line height around long-term. See for example [`line-height: normal`](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#normal), for a case where this does change based on the font family.
The other arguments which I think can be dismissed:
- CSS overloads the `line-height` property to mean both
- It's a pretty bad breaking change

There is another argument, which is one of defaults: any default absolute line height is stupid.

Reply:
> Is a relative line height ever affected by the selected font (not just now, but is that something we might theoretically ever want?). That to me would be the only reasonable argument for keeping a relative line height around long-term.

It's not now in Parley, but it used to be (and I suspect this is why Parley currently uses relative line height). It was changed in https://github.com/linebender/parley/pull/84/files to match CSS (which defines line height relative to the nominal font size). Previously it took into account the metrics from the font.

Whether this is ever desirable I'm not sure.